src/migrateDashboard.test.ts

```diff
@@ -237,4 +253,39 @@ describe("migrateDashboard", () => {
       }
     `);
   });
+
+  it("removes the specified widget keys, and adapts the layout", () => {
+    const keysOfWidgetPluginsToRemove = ["filters"];
+
+    const dashboard = migrateDashboard(
+      legacyDashboard,
+      servers,
+      keysOfWidgetPluginsToRemove
+    );
+
+    const { content, layout } = dashboard.pages["p-0"];
+    const widgetPluginKeys = _map(content, ({ widgetKey }) => widgetKey);
+    expect(
```

Comment:
It would be nice to add another assertion here with the legacy dashboard state:

```
expect(
  _intersection(widgetKeysInLegacyDashboard, keysOfWidgetPluginsToRemove)
).toHaveLength(1);
```

This way, we are guaranteed that this test will remain useful, even if future devs change the content of mocks.